fuuk/people/models/grant.py

```diff
@@ -22,31 +25,62 @@ def __unicode__(self):
 
 
 class Grant(models.Model):
-    author = models.ForeignKey(Person, verbose_name=_('Applicant'))
-    number = models.CharField(max_length=20)
-    start = models.SmallIntegerField(validators=[MinValueValidator(1990), MaxValueValidator(date.today().year + 1)])
-    end = models.SmallIntegerField(validators=[MinValueValidator(1990), MaxValueValidator(date.today().year + 10)])
-    co_authors = models.ManyToManyField(Person, related_name='grant_related', blank=True,
-                                        verbose_name=_('Co-applicant'))
-    agency = models.ForeignKey(Agency, help_text=_('Contact administrators for different Grant Agency.'))
+    editor = models.ForeignKey(User, blank=True, null=True, on_delete=models.SET_NULL, verbose_name=_('editor'))
+    investigator_prefix = models.CharField(_('prefix'), max_length=20, blank=True, null=True)
+    investigator_first_name = models.CharField(_('first name'), max_length=50)
+    investigator_last_name = models.CharField(_('last name'), max_length=50)
+    investigator_suffix = models.CharField(_('suffix'), max_length=20, blank=True, null=True)
+    investigator_institution = models.ForeignKey(Institution, blank=True, null=True, on_delete=models.CASCADE,
+                                                 verbose_name=_('institution'))
+    investigator_human = models.ForeignKey(Human, blank=True, null=True, on_delete=models.SET_NULL,
+                                           verbose_name=_('human'))
+    number = models.CharField(_('grant number'), max_length=20)
+    start = models.SmallIntegerField(_('start year'), validators=[MinValueValidator(1990),
+                                     MaxValueValidator(date.today().year + 1)])
```

Comment:
There is one hidden problem in here. The `today()` is evaluated when process starts, and it may run for few months, which means the calendar year may change. The only way to prevent this is to put year in lazy evaluation.